File: test-suite-kotlin/src/test/kotlin/io/micronaut/docs/server/sse/Headline.kt
package io.micronaut.docs.server.sse

// tag::class[]
class Headline {
    var title: String? = null
    var description: String? = null

    constructor() {}

    constructor(title: String, description: String) {
        this.title = title
        this.description = description
    }
}
// end::class[]

File: test-suite-python/src/test/python/micronaut/docs/server/sse/Headline.py
from dataclasses import dataclass

from micronaut.core.annotation import Introspected, ReflectiveAccess


# tag::class[]
@ReflectiveAccess
@Introspected
@dataclass
class Headline:
    title: str | None = None
    description: str | None = None

    def getTitle(self) -> str | None:
        return self.title

    def getDescription(self) -> str | None:
        return self.description

    def setTitle(self, title: str) -> None:
        self.title = title

    def setDescription(self, description: str) -> None:
        self.description = description
# end::class[]
